api/tests/core/finance/test_api.py

```diff
@@ -2292,7 +2292,6 @@ def test_create_18_years_old_deposit(self):
         assert deposit.expirationDate == datetime.datetime(2024, 9, 5, 21, 59, 59, 999999)
 
     @freezegun.freeze_time("2022-09-05 09:00:00")
-    @override_settings(IS_INTEGRATION=True)
```

Comment:
(détail) cette ligne supprimée devrait appartenir à l'autre commit ("[refactor: remove obsolete ref to the environment](https://github.com/pass-culture/pass-culture-main/pull/6134/commits/f4856cb4822c8b96b4045ae5b69de0b16615ffd5)").

(micro-détail) On pourrait en profiter pour renommer le test, puisqu'il n'est plus spécifique à l'environnement d'intégration. Ou peut-être même qu'on devrait le supprimer car il semble identique à `test_create_18_years_old_deposit()` juste au-dessus, non ?